packages/binding-http/README.md

```diff
@@ -226,6 +227,43 @@ Currently this binding supports only oAuth2.0 `client credential` and `Resource
 
 An example of a WoT oAuth2.0 enabled client can be found [here](../examples/security/oauth).
 
+
+### Using BaseUri
+
+> BaseUri allows the TD to use hostnames, network interfaces, and URI prefixes whicch are not local to the machine exposing the Thing
+
+Assume the coffee machine example is sitting in an office inside the W3C's private network.  It allows you to start the coffee machine before you leave home so it will be ready when you get to work.
+
+* It can be addressed from home via the internet facing domain name `https://wot.w3c.org/things/smart-coffee-machinet`
+* The actual coffee machine TD is at `https://internal-host:8080/coffee-pot`
+
+__HttpServer Configuration__
+```js
+servient.addServer(new HttpServer({
+    port: 8080, // (default 8080)
+    baseUri: 'https://coffee.w3c.org/things'
+}));
+```
+
+__External Gateway Configuration__
+> assume nginx
+
+```
+location /things/ {
+    proxy_pass https://internal-host:8080/smart-coffee-machine
+}
+```
+
+The exposed thing on the internal server will product form URLs such as:
+
+```json 
+ "actions": {
+    "makeDrink": {
+      "forms": [
+        {
+          "href": "https://wot.w3c.org/pot/things/smart-coffee-machine/actions/makeDrink"
```

Comment:
Should it be: "https://coffee.w3c.org/things/smart-coffee-machine/actions/makeDrink" ?